### PR TITLE
Thahir quick corrections

### PR DESCRIPTION
**Pull request by AhmedThahir**
- Removed labels for fields; Placeholder is sufficient
- Changed registration button from Technofest to School, Enginuity to University (people may not remember which is which)
- Compressed images; 3mb for an image on a website is ridiculous, guys come on.
- Made mobile number format clearer in the placeholder
- Removed unnecessary items to make it easier to register

### CODE REVIEW

File: src/components/Pages/AboutUs.js [hunk 1]
```diff
@@ -8,30 +8,32 @@ import icon3 from '../themes-icons/AI.png';
...
+      <div className='about-us' style={{ backgroundImage: `url(${Img})` }}>
         <h1>About BITS Tech Fest</h1>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed iaculis venenatis blandit. Quisque et consectetur ligula. Cras imperdiet rutrum diam, lacinia sagittis tortor viverra quis. Mauris vitae molestie mi. Vestibulum id tincidunt libero. Suspendisse a tempus magna. Nullam mattis, leo non fermentum posuere, massa lacus lacinia turpis, non convallis odio sem a augue. Morbi sollicitudin massa et blandit imperdiet. Interdum et malesuada fames ac ante ipsum primis in faucibus. Mauris id vehicula felis. Morbi vulputate risus vel semper accumsan. Morbi placerat accumsan rhoncus. Mauris ac suscipit ipsum, lacinia ultricies elit.</p>
-        <br/>
-        <p>Save the date- May 1 and 2, 2024</p> <br/>
+        <p>BITS Tech Fest is BITS Pilani Dubai's exclusive annual tech festival with students from all over UAE.</p> <br />
+        <p>Technofest is for the next generation of aspiring students to explore the world of engineering.</p> <br />
+        <p>Enginuity is for engineering students to showcase their skills.</p> <br />
```
Comment (howdoiusekeyboard) [inline src/components/Pages/AboutUs.js:15]: Isn't Enginuity Techno-Manegorial?, so not just engineering students

File: src/components/Pages/AboutUs.js [hunk 2]
```diff
@@ -8,30 +8,32 @@ import icon3 from '../themes-icons/AI.png';
...
   return (
     <>
-     <div className='about-us' style={{ backgroundImage:  `url(${Img})` }}>
+      <div className='about-us' style={{ backgroundImage: `url(${Img})` }}>
         <h1>About BITS Tech Fest</h1>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed iaculis venenatis blandit. Quisque et consectetur ligula. Cras imperdiet rutrum diam, lacinia sagittis tortor viverra quis. Mauris vitae molestie mi. Vestibulum id tincidunt libero. Suspendisse a tempus magna. Nullam mattis, leo non fermentum posuere, massa lacus lacinia turpis, non convallis odio sem a augue. Morbi sollicitudin massa et blandit imperdiet. Interdum et malesuada fames ac ante ipsum primis in faucibus. Mauris id vehicula felis. Morbi vulputate risus vel semper accumsan. Morbi placerat accumsan rhoncus. Mauris ac suscipit ipsum, lacinia ultricies elit.</p>
-        <br/>
-        <p>Save the date- May 1 and 2, 2024</p> <br/>
+        <p>BITS Tech Fest is BITS Pilani Dubai's exclusive annual tech festival with students from all over UAE.</p> <br />
+        <p>Technofest is for the next generation of aspiring students to explore the world of engineering.</p> <br />
```
Comment (howdoiusekeyboard) [inline src/components/Pages/AboutUs.js:14]: Are you sure speciating to just engineering?

**Comment by howdoiusekeyboard** [comment]
Hii, thanks for the PR

1. I think we should remove the placeholder if its repetitive instead of the labels because someone who has filled in their inputs might get confused at the end when they want to make a quick cross check of their data before submitting. As for the placeholder either it be blank for provide a specific instruction to the label

2. We could put schools and University in brackets ; eg Technofest (School). That would be better because at the end we don't want to lose oppurtunities on branding technofest and Enginuity

3. Yes good move. Compressed images would help with load times

4. Accepted

5. LGTM

Note: Edits to title img have disaligned it, it looked better before. Please fix

**Review by howdoiusekeyboard — CHANGES_REQUESTED** [review]

File: src/components/Pages/AboutUs.js (hunk 1 above)
Comment (AhmedThahir) [reply src/components/Pages/AboutUs.js:15]: You're right.

File: src/components/Pages/AboutUs.js (hunk 2 above)
Comment (AhmedThahir) [reply src/components/Pages/AboutUs.js:14]: Right again

**Comment by AhmedThahir** [comment]
Mahn this vercel needs so much authentication just for preview. So irritating. I just want to see the preview. Is there an easier method?

**Pull request merged**